### ISSUE TITLE
/skip command is not working as expected

### ISSUE THREAD

**Issue by justinsb** [kubernetes/test-infra#24123]
Similar to #12740, it's not clear what the skip command is for.

Context is https://github.com/kubernetes/kops/pull/12382#issuecomment-951873683.  We have a test that doesn't run automatically.  We do `/test foo` to trigger it.  It fails, but we then agree we're going to address the failure separately, so we want to `/skip foo`.  However, `/skip foo` doesn't undo `/test foo`.

Apparently, per #12740, we have to mark the tests as optional.  But if we mark the tests as optional, https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#requiring-jobs-for-auto-merge-through-tide says that the test is ignored for merging.  So I don't think we want it to be `optional: true`: If we invoke a test, we want it to block merge unless we decide it's OK for it to fail (which should be the exception).

In other words, if the skip command can only skip tests that are disregarded anyway, it's not clear what the skip command is for.

**Comment by jmguzik** [comment]
@justinsb I am not an expert here, but the skip plugin is described this way: The skip plugin allows users to clean up GitHub stale commit statuses for non-blocking jobs on a PR.
I would see an /override use-case here, after failing with the /test.
